README.md

```diff
@@ -105,7 +105,15 @@ Choice must also be `<None>`.
     }
 }
 ```
-
+### Using Tyr as a Library
+
+| Api                                               | Description                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Below you should use the code formatting. 

For example `async generateProject(configs)`